src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php

```diff
@@ -111,15 +111,6 @@ public function createAuthenticator(ContainerBuilder $container, string $firewal
             ->replaceArgument(3, $config['name'] ?? $this->options['name'])
         ;
 
-        foreach ($container->findTaggedServiceIds('security.remember_me_aware') as $serviceId => $attributes) {
-            // register ContextListener
-            if (str_starts_with($serviceId, 'security.context_listener')) {
-                continue;
-            }
-
-            throw new \LogicException(sprintf('Symfony Authenticator Security dropped support for the "security.remember_me_aware" tag, service "%s" will no longer work as expected.', $serviceId));
```

Comment:
@wouterj Do you have any idea why we didn't just remove that code in 6.0? It feels like it was on purpose

Reply:
I think we've had a badly resolved merge conflict in this part. But this tag appears to be so underused, that we never received an issue about it :)

This code contained the correct logic in 5.1 (when the security rework was done) and 5.2: https://github.com/symfony/symfony/blob/5.1/src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php#L114-L126

The legacy logic in 5.3 and 5.4: https://github.com/symfony/symfony/blob/5.3/src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php#L64-L84

And finally, the current state without any real logic in 6.0, where it should just be completely removed (As it doesn't do anything anymore).